specification/motivation.md

```diff
@@ -0,0 +1,16 @@
+Motivation
+=========
+
+Although much progress has been made in standardizing neuroimaging pipelines, model fitting requires a high degree of flexibility, resulting in heterogeneous scripts that are often not **reproducible**. Upon publication, models are often verbally described, making it difficult to reproduce analyses from published studies. Even when analysis scripts are shared, they often lack **transparency**, making it difficult for other researchers to interpret and apply the same model to a different dataset. 
+
+Goal
+--------- 
+
+*BIDS Stats Models* aims to facilitate the development **automated and standardized** neuroimaging model fitting pipelines, encouraging **reproducible and generalizable scientific** practice. In addition, by promoting the development of an ecosystem of easy to use and **interoperable** tools, BSM aims to facilitate the process of building and fitting statistical models on neuroimaging data.
```

Comment:
I would also add the definition of the abbreviation because it is not yet defined in this section. So "*BIDS Stats Models (BSM)* aims to ...."

Reply:
I made a mental note regarding this the other day but forgot to report on it: in general I would avoid adding the abbreviation at all. If someone jumped in the middle of the walkthrough (because we send them a link to that specific section), they will now how have to go to the beginning of the whole thing to figure out what the abbreviation sands for.

I can open a PR to make a general change on this.

Reply:
That's fair, I was actually going back an forth on this.

Alternative what is the abbreviation is "BIDS-SM". That seems at least more obvious.

Reply:
Will open a cleaning PR:
- remove BSM abbreviation
- enforce hard word wrapping